openvpn/middlewares/server/auth/middleware.go

```diff
@@ -170,8 +173,10 @@ func (m *middleware) authenticateClient() (consumed bool, err error) {
 
 	log.Info("authenticating user: ", m.lastUsername, " clientID: ", m.clientID, " keyID: ", m.keyID)
 
-	authenticated, err := m.authenticator(m.lastUsername, m.lastPassword)
+	authenticated, err := m.checkCredentials(m.lastUsername, m.lastPassword)
 	if err != nil {
+		log.Error("Authentication error: ", err)
+		denyClientAuthWithMessage(m.connection, m.clientID, m.keyID, "internal error")
 		return false, err
```

Comment:
Do we really need bad crediantials reason as queue consuming error?

Reply:
Improved